### ISSUE TITLE
[DataObjects] Multiple Deprecation Warnings

### ISSUE THREAD

**Issue by fashxp** [pimcore/pimcore#3778]
```
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\Textarea uses method Pimcore\Model\DataObject\ClassDefinition\Data::setColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\ManyToOneRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setQueryColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\ManyToOneRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\ManyToManyObjectRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setQueryColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\ManyToManyObjectRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\Image uses method Pimcore\Model\DataObject\ClassDefinition\Data::setQueryColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\Image uses method Pimcore\Model\DataObject\ClassDefinition\Data::setColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\ManyToOneRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setQueryColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\ManyToOneRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\ManyToManyObjectRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setQueryColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\ManyToManyObjectRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\AdvancedManyToManyRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setQueryColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\AdvancedManyToManyRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\AdvancedManyToManyObjectRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setQueryColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\AdvancedManyToManyObjectRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\ManyToManyRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setQueryColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
PHP Deprecated: Pimcore\Model\DataObject\ClassDefinition\Data\ManyToManyRelation uses method Pimcore\Model\DataObject\ClassDefinition\Data::setColumnType from the abstract class. This won't work in v6.0, please use the proper interfaces and provided traits. in /home/ecommerce/demo-ecommerce/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data.php on line 1389
```

**Comment by blankse** [comment]
The file paths seem to be old. Should not a vendor folder be found here?

**Comment by fashxp** [comment]
connected with this one here https://github.com/pimcore/pimcore/pull/3747 and this commit https://github.com/pimcore/pimcore/commit/0c0fd117d82cddc7585fc2ef755fa94ad0a190d9 

warnings are triggered, when class definition files are included and `__set_state` is called.

**Comment by brusch** [comment]
fixed by #3789
